src/modules/dashboard/tabReviewInvites/sendReviewInvitesForPreviousOrders.tsx

```diff
@@ -66,12 +71,21 @@ const SendReviewInvitesForPreviousOrders: FC<Props> = ({
               id={'exportCSV'}
               label={phrasesByKey.application_invites_send_export_button}
               theme={ButtonThemes.Primary}
-              onClick={() =>
+              onClick={() => {
                 onExport(
                   selectedShopChannels?.eTrustedChannelRef,
-                  selectedShopChannels.salesChannelRef
+                  selectedShopChannels.salesChannelRef,
                 )
-              }
+                try {
+                  postEtrustedInteractions(user?.access_token as string, {
+                    interaction: InteractionType.DATA_EXPORTED,
+                    allState,
```

Comment:
Can I see exported data inside of allState?

Reply:
I checked but we don't have that information that happens on the fly on the baseLayer side we are only invoking the functionality of export method but without any information about the data of the exported CSV